src/webui/api/torrentscontroller.cpp

```diff
@@ -1038,3 +1038,78 @@ void TorrentsController::categoriesAction()
 
     setResult(categories);
 }
+
+void TorrentsController::addTagsAction()
+{
+    checkParams({"hashes", "tags"});
```

Comment:
Although I approve "hashes" as param name since the rest of the methods historically use this name, but I would personally prefer to call it "torrents".